### PR TITLE
Added Coingecko price script. .env.example for the validator

### PR DESCRIPTION
**Pull request by Abhishekkochar**

### CODE REVIEW

File: oracle/index.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+const {coingeckoPrice} = require('./collector')
+const nodeCron = require("node-cron");
 require('dotenv').config()
 
 async function main() {
     const data = await collect();
     console.log('collection', JSON.stringify(data, null, '\t'));
-
+    //Relay script will run first second of first minute every hour
+    await nodeCron.schedule(" 1 1 * * * *",async function(){
```
Comment (gophereth) [inline oracle/index.js:10]: not only price relay function to be executed every hour, whole script should be executed every hour.

File: oracle/collector/coingecko/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const {priceFeed} = require('./priceScript');
+
+async function cronPriceJob(){
+    const priceJob = await priceFeed();
```
Comment (gophereth) [inline oracle/collector/coingecko/index.js:4]: I don't see priceFeed implementation

File: oracle/index.js (hunk 1 above)
Comment (gophereth) [reply oracle/index.js:10]: Price and reward information should be relayed per hour

**Comment by gophereth** [comment]
I don't see .env.example added here

**Comment by gophereth** [comment]
And should fix merge conflicts with main and for the branch - we should use new branch rather than using old one `ethPolyProtocol`

File: oracle/collector/coingecko/index.js (hunk 2 above)
Comment (Abhishekkochar) [reply oracle/collector/coingecko/index.js:4]: Seemed like the Price script didn't commit to this brack. It should be now.

File: oracle/index.js (hunk 1 above)
Comment (Abhishekkochar) [reply oracle/index.js]: Done

File: oracle/collector/index.js [hunk 3]
```diff
@@ -3,20 +3,30 @@ const { collect_osmosis } = require('./osmosis');
 const { collect_solana } = require('./solana');
 const { collect_eth } = require('./ethereum');
 const { collect_polygon } = require('./polygon')
+const { cronPriceJob } = require('./coingecko')
```
Comment (gophereth) [inline oracle/collector/index.js:6]: Let's not call it cronPriceJob - let's just call it `collect_price` similar to other function names

File: oracle/collector/index.js [hunk 4]
```diff
@@ -3,20 +3,30 @@ const { collect_osmosis } = require('./osmosis');
...
     return {
         bsc,
         solana,
         osmosis,
         eth,
         poly,
+        coingecko,
     }
 }
+//  async function coingeckoPrice(){
```
Comment (gophereth) [inline oracle/collector/index.js:25]: Unused code can be removed

**Pull request merged**